flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTest.java

```diff
@@ -368,17 +368,19 @@ void testCleanUpExceptionSuppressing() throws Exception {
                         .setupOutputForSingletonOperatorChain(new FailingTwiceOperator())
                         .build()) {
 
-            assertThatThrownBy(
-                            () -> {
-                                testHarness.processElement(new StreamRecord<>("Doesn't matter", 0));
-                                throw new RuntimeException(
-                                        "Expected an exception but ran successfully");
-                            })
-                    .isInstanceOf(ExpectedTestException.class);
+            try {
+                testHarness.processElement(new StreamRecord<>("Doesn't matter", 0));
+                throw new RuntimeException("Expected an exception but ran successfully");
+            } catch (Exception ex) {
+                ExceptionUtils.assertThrowable(ex, ExpectedTestException.class);
+            }
```

Comment:
```suggestion
            Assertions.assertThatThrownBy(
                            () ->
                                    testHarness.processElement(
                                            new StreamRecord<>("Doesn't matter", 0)))
                    .satisfies(
                            throwable ->
                                    ExceptionUtils.assertThrowable(
                                            throwable, ExpectedTestException.class));
```

Reply:
nice.
thx so much for the quick comments.